dev-packages/node-integration-tests/suites/tracing/amqplib/utils.ts

```diff
@@ -0,0 +1,38 @@
+import amqp from 'amqplib';
+import type { Channel, Connection } from 'amqplib';
+import { ACKNOWLEDGEMENT, AMQP_URL, QUEUE_OPTIONS } from './constants';
+
+export type RabbitMQData = {
+  connection: Connection;
+  channel: Channel;
+};
+
+export async function connectToRabbitMQ(): Promise<RabbitMQData> {
```

Comment:
m: I would generally inline all of these into the scenario file, unless we have multiple scenarios that need to share this. If we only have a single scenario file (which is often the case) I would try to put all the scenario code into the scenario file, this makes it easier to parse tests etc. The same goes for constants, I'd also merge this into the file :)

Reply:
we might have more, I had some problem with testing one scenario and then just give up. But even though I like to keep things separated so it is easier to see things. Would rather avoid keeping all inside one file, WDYT ?

Reply:
no strong feelings! I just generally would avoid premature optimization, we can always extract stuff out when we actually _need_ a second file, but also I won't block this as it is now, it's OK like this too, just wanted to give general feedback :)